pxd.c

```diff
@@ -181,6 +181,37 @@ static long pxd_ioctl_resize(struct file *file, void __user *argp)
 	return ret;
 }
 
+static long pxd_ioctl_fp_cleanup(struct file *file, void __user *argp)
```

Comment:
a new ioctl to release volume in suspended state after attach failed in fastpath.